apps/backend/file-service/src/test/resources/application-test.yml

```
@@ -15,4 +15,7 @@ spring:
       ddl-auto: update
   sql:
     init:
-      mode: always
\ No newline at end of file
+      mode: always
+      schema-locations:
+        - classpath:schema.sql
+        - classpath:functions/*.sql
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)